css/main.css

```diff
@@ -130,6 +131,16 @@ body {
   font-size: 12px;
 }
 
+#kanjiSelectionBox h4 {
+  font-weight: bold;
+  text-transform: uppercase;
+  font-size: 10px;
+}
+
+#kanjiSelectionBox .category-select-all, .category-deselect-all {
```

Comment:
It would be great if these could be stacked on top of each other since they aren't all part of one selector.

Also as is the `.category-deselect-all` isn't using the parent selector `#kanjiSelectionBox`
```
#kanjiSelectionBox .category-select-all,
#kanjiSelectionBox .category-deselect-all
```